snmp/tests/test_profiles.py

```diff
@@ -1718,6 +1718,12 @@ def test_apc_ups(aggregator):
         'upsAdvOutputLoad',
         'upsBasicBatteryTimeOnBattery',
         'upsAdvTestDiagnosticsResults',
+        'upsBasicStateOutputState.AVRTrimActive',
+        'upsBasicStateOutputState.BatteriesDischarged',
+        'upsBasicStateOutputState.LowBatteryOnBattery',
+        'upsBasicStateOutputState.NoBatteriesAttached',
+        'upsBasicStateOutputState.OnLine',
+        'upsBasicStateOutputState.ReplaceBattery',
```

Comment:
Shall test the value for those metrics ? it's quite important for this case I think.